packages/labs/compiler/src/lib/type-checker.ts

```diff
@@ -93,9 +93,18 @@ class TypeChecker {
       return false;
     }
 
+    // An import specifier has the following structure:
+    //
+    // `import {<propertyName> as <name>} from <moduleSpecifier>;`
+    //
+    // This check allows aliasing `html` by ensuring propertyName is `html`.
+    // Thus `{html as myHtml}` is a valid template that can be compiled.
+    // Otherwise a compilable template must be a direct import of lit's `html`
+    // tag function.
     if (
-      templateImport.propertyName &&
-      templateImport.propertyName.text !== 'html'
+      (templateImport.propertyName &&
+        templateImport.propertyName.text !== 'html') ||
+      (!templateImport.propertyName && templateImport.name.text !== 'html')
```

Comment:
does this mean that in the case of no alias, the import specifier is like this?
```
import {<name>} from <moduleSpecifier>;
```
so if there is no `propertyName`, we check that `name` is the right value?

Reply:
That is exactly right, I added your example without the alias to the source code.